msteams-platform/m365-apps/prerequisites.md

```diff
@@ -84,6 +86,27 @@ To install Office 365 Beta Channel applications in your test environment:
 
     If it's not listed, install [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/) to your test environment.
 
+### Mobile
+
+You can preview Teams personal tabs running in Office app for Android by joining the beta program.
+
+To install the latest Office app beta build to your physical Android device or Android emulator:
+
+1. Verify you are using a Google Play [supported Android device](https://support.google.com/googleplay/answer/1727131).
+1. On your Android device, launch the **Play Store**.
+1. Search for office and select **Microsoft Office: Edit & Share**.
+1. Select the **Install** button.
+
+    :::image type="content" source="images/office-android-install.png" alt-text="screenshot of Microsoft Office: Edit & Share app in Google Play Store":::
+
+1. Once installed, from the same screen under Join the beta select **Join**.
```

Comment:
Please change to:
Once installed, select **Join** under **Join the beta section**.